third_party/txt/src/txt/platform_linux.cc

```diff
@@ -4,24 +4,16 @@
 
 #include "txt/platform.h"
 
-#ifdef FLUTTER_USE_FONTCONFIG
-#include "third_party/skia/include/ports/SkFontMgr_fontconfig.h"
-#else
-#include "third_party/skia/include/ports/SkFontMgr_directory.h"
-#endif
-
 namespace txt {
 
 std::vector<std::string> GetDefaultFontFamilies() {
   return {"Ubuntu", "Cantarell", "DejaVu Sans", "Liberation Sans", "Arial"};
 }
 
 sk_sp<SkFontMgr> GetDefaultFontManager(uint32_t font_initialization_data) {
-#ifdef FLUTTER_USE_FONTCONFIG
-  return SkFontMgr_New_FontConfig(nullptr);
-#else
-  return SkFontMgr_New_Custom_Directory("/usr/share/fonts/");
-#endif
+  // WARNING: Not using `SkFontMgr::RefDefault()` can cause performance issues
```

Comment:
Remove this comment - using `SkFontMgr::RefDefault` does not need a specific explanation.  `RefDefault` is the standard choice unless the platform has a specific reason to override it.

(IIRC the previous implementation was needed because at that time Skia did not ensure that the FontConfig font manager would be preferred over the directory font manager if both are available)

Reply:
Should be fixed now.